stylesheets/components/Image/images.css

```diff
@@ -0,0 +1,22 @@
+    /* Responsive Image */
+    .image--responsive {
+        max-width: 100%;
+        object-fit: cover;
+      }
+  
+      .image--round img {
+        border-radius: 50%;
+        width: 200px;
+        height: 200px;
+      }
+  
+      .image--square img {
+        width: 200px;
+        height: 200px;
```

Comment:
same as above

Reply:
Thank You Nakul. Corrected in the latest commit.